parser-typechecker/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -523,15 +523,13 @@ loop = do
           :: SlurpComponent v
           -> Action m (Either Event Input) v ()
         addDefaultMetadata adds = do
-          let
-            addedVs = Set.toList $ SC.types adds <> SC.terms adds
-            parseResult = Path.parseHQSplit' . Var.nameStr <$> addedVs
-            (errs, addedNames) = partitionEithers parseResult
-          case errs of
-            e : _ ->
+          let addedVs = Set.toList $ SC.types adds <> SC.terms adds
+              addedNs = traverse (Path.hqSplitFromName' . Name.fromVar) addedVs
```

Comment:
The main change here is the switch from `parseHQSplit'` to `hqSplitFromName'` which doesn't try to parse a `Path` by itself. The name here will never be just a path.